src/backend/distributed/shared_library_init.c

```diff
@@ -217,6 +218,13 @@ static const struct config_enum_entry explain_analyze_sort_method_options[] = {
 	{ NULL, 0, false }
 };
 
+
+static const struct config_enum_entry executor_connection_strategy[] = {
+	{ "none", EXECUTOR_CONNECTION_STRATEGY_NONE, false },
```

Comment:
wondering whether we can give this a name, e.g. slow-start-only

Reply:
Maybe change this to a hidden feature flag and we just treat the new behaviour as "the" behaviour.